app/controllers/public/exhibition/view.js

```diff
@@ -40,6 +41,19 @@ export default class extends Controller {
     return this.session.isAuthenticated;
   }
 
+  @action
+  changeExhibitor(flag) {
+    let nextPos = this.model.exhibitor.position + flag;
+    if (nextPos < 0) {
+      nextPos = this.model.exhibitors.toArray().length - 1;
+    }
+    if (nextPos === (this.model.exhibitors.toArray().length)) {
+      nextPos = 0;
+    }
+    const nextExhibitor = this.model.exhibitors.toArray().filter(exh => exh.position === nextPos);
```

Comment:
here if exhibitors are not ordered, all of them have a position of 0.
So shouldn't server by default instead of storing position as 0, store it as the next integer?

Reply:
next integer to what?

Reply:
make position as auto increment column?